tests/system/robot/chromeTests.py

```diff
@@ -140,6 +140,125 @@ def test_mark_aria_details_FreeReviewCursor():
 	exercise_mark_aria_details()
 
 
+def test_mark_aria_details_role():
+	_chrome.prepareChrome(
+		"""
+		<div class="editor" contenteditable spellcheck="false" role="textbox" aria-multiline="true">
+			<p>
+				<span aria-details="endnote-details">doc-endnote,</span>
+				<span aria-details="footnote-details">doc-footnote,</span>
+				<span aria-details="comment-details">comment,</span>
+				<span aria-details="definition-details">definition,</span>
+				<span aria-details="unknown-details">form</span>
+			</p>
+		</div>
+		<div>
+			<p>
+				<div id="endnote-details" role="doc-endnote">details with role doc-endnote</div>
+				<div id="footnote-details" role="doc-footnote">details with role doc-footnote</div>
+				<div id="comment-details" role="comment">details with role comment</div>
+				<div id="definition-details">details with role definition</div>

```

Comment:
This doesn't actually have a role, further it doesn't seem to match examples for how this should work.
See:
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dfn
- https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/definition_role

It would be good to check the other items too.

Reply:
yes this was a mistake when submitting code. Locally, I have tested both `<definition>` and `<definition role="definition">`. Both of these report the wrong role, both on the accessibility tree and in NVDA.